src/core/formatters.ts

```diff
@@ -60,7 +60,7 @@ export module Formatters {
    * @returns {Formatter} A formatter that displays exactly [precision] decimal places.
    */
   export function fixed(precision = 3) {
-    verifyPrecision(precision);
+    precision = verifyAndAdjustPrecision(precision);
```

Comment:
Agree with second comment, and I think I agree with the first comment.

If we do go with essentially a sanitizing idea, maybe `sanitizePrecision()`?  It does not signal well if the input were to error though.